tests/build.go

```diff
@@ -247,6 +247,10 @@ RUN busybox | head -1 | grep v1.36.1
 }
 
 func testBuildDetailsLink(t *testing.T, sb integration.Sandbox) {
+	if isExperimental() {
+		t.Skip("build details link not displayed in experimental mode")
+	}
```

Comment:
Build details is not displayed when building through controller: https://github.com/docker/buildx/actions/runs/8522265808/job/23342157852?pr=2376#step:7:771

Needs to investigate, not sure if this is a regression